src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -126,6 +126,34 @@ def detect_performance_problems(data: dict[str, Any], project: Project) -> List[
     return []
 
 
+def get_performance_issues_project_settings(
+    project_id: Optional[int] = None,
+) -> Dict[str | Any, Any]:
+    default_project_settings = (
+        projectoptions.get_well_known_default(
+            "sentry:performance_issue_settings",
+            project=project_id,
+        )
+        if project_id
+        else {}
+    )
+
+    project_option_settings = (
+        ProjectOption.objects.get_value(
+            project_id, "sentry:performance_issue_settings", default_project_settings
+        )
+        if project_id
+        else DEFAULT_PROJECT_PERFORMANCE_DETECTION_SETTINGS
+    )
+
+    project_settings = {
+        **default_project_settings,
+        **project_option_settings,
+    }  # Merge saved project settings into default so updating the default to add new settings works in the future.
+
+    return project_settings
+
```

Comment:
isolated project settings retrieval into a function to be reused in the stats detector task